scene/resources/curve.h

```diff
@@ -272,6 +285,14 @@ class Curve3D : public Resource {
 	static void _bind_methods();
 
 public:
+	enum RotationMode {
+		ROTATION_NONE,
+		ROTATION_Y,
+		ROTATION_XY,
+		ROTATION_XYZ,
+		ROTATION_ORIENTED
+	};
```

Comment:
```suggestion
```
Now I know the cause of the weirdness. This enum should not have at least Curve3D, because `Curve3D::set_rotation_mode` does not exist.